.golangci.yml

```diff
@@ -0,0 +1,14 @@
+linters:
+  disable-all: true
```

Comment:
Why?

Reply:
I wanted validation that we don't add back the dependency but this repo does not yet have linting. I can draft a commit for adding other linters. Most are trivial, but errcheck has a few spots I was not sure about.